Formula/ice.rb

```diff
@@ -58,6 +58,7 @@ def install
     args = %W[
       prefix=#{prefix}
       USR_DIR_INSTALL=yes
+      SLICE_DIR_SYMLINK=yes
```

Comment:
The build system changed how Ice's "slice" files are installed. We (I'm an Ice developer) changed the behaviour so that by default there is a now globally installed directory for all users to install their slice files (`/PREFIXPATH/share/slice`). Previously it was just a simlink to the slice folder (which is installed in a different location). 

Since Homebrew linking creates simlinks, and only special formula can create directories (bash, fish, etc.),  this uses the "old" logic and just creates a simlink.